libraries/AP_NavEKF3/AP_NavEKF3_RngBcnFusion.cpp

```diff
@@ -23,7 +23,7 @@ void NavEKF3_core::SelectRngBcnFusion()
     // Determine if we need to fuse range beacon data on this time step
     if (rngBcnDataToFuse) {
         if (PV_AidingMode == AID_ABSOLUTE) {
-            if (!filterStatus.flags.using_gps) {
+            if (!filterStatus.flags.using_gps && rngBcnAlignmentCompleted) {
```

Comment:
This change will have the effect that the EKF will switch to PV_AidingMode = AID_ABSOLUTE but will not fuse any data  to constrain horizontal drift until beacon alignment is complete. If you are trying to prevent the use of range beacon data before the beacon origin estimate has stabilised, It would be better to not defer entry into PV_AidingMode = AID_ABSOLUTE until the alignment is complete.

I can have a look at that if the SITL support for range beacons is still working.

Reply:
See the commit I have added here: https://github.com/priseborough/ardupilot/tree/pr7712

I will now try to get the range beacon sim working in SITL

Reply:
I was able to get it to initialise and start using beacon data without GPS using these parameters: 
[sitl_bcn.txt](https://github.com/ArduPilot/ardupilot/files/1853982/sitl_bcn.txt)

Reply:
this was because we get a missing condition : If we don't have gps and use beacon is directly switch to PV_AidingMode == AID_ABSOLUTE and then cannot FuseRngBcnStatic() that is need to make the alignment.
I think your patch is better done ! Thanks for looking into this

Reply:
So this change still is in the PR and I'm not sure if @priseborough thinks that's OK or not. @priseborough can you clarify?

Reply:
I did my patch on top of https://github.com/ArduPilot/ardupilot/pull/7712/commits/1de08aa9bd694a13db03570aa4ab6860269fde2a so it is OK to leave it in

Reply:
Sorry to be a pain @priseborough, but I want to really make sure I'm not going to merge it wrongly. Your patch **isn't** reverting this change, is that correct?